run.py

```diff
@@ -919,79 +919,86 @@ def alignments(args):
       genomeid_to_taxid = json.load(inf)
 
    for sample in get_samples(args):
-      combined_output_compressed = sample + ".alignments.tsv.gz"
-      if combined_output_compressed in existing_outputs:
-         continue
+      for db in ["human", "hv"]:
+         combined_output_compressed = "%s.%.alignments.tsv.gz" % (
```

Comment:
Should there be a second s in front of the second %? i.e, `"%s.%s.alignments.tsv.gz"`

Reply:
Good catch; fixed!